key.core/src/main/antlr4/JmlParser.g4

```diff
@@ -279,7 +279,84 @@ primaryexpr
   | array_initializer
   ;
 this_: THIS;
-ident: IDENT | JML_IDENT | SPECIAL_IDENT | THIS | SUPER;
+
+ident: IDENT | JML_IDENT | SPECIAL_IDENT | THIS | SUPER
+	    MODEL_BEHAVIOUR |         BEHAVIOR |
+        EXCEPTIONAL_BEHAVIOUR |
+       BREAK_BEHAVIOR|        CONTINUE_BEHAVIOR|
+       ALSO|        CODE_BIGINT_MATH|
+       CODE_JAVA_MATH|
+       CODE_SAFE_MATH|
+       CONST|        NATIVE|
+       NON_NULL|        NORMAL_BEHAVIOR|
+       NO_STATE|       NOWARN|
+       NULLABLE|        NULLABLE_BY_DEFAULT|
+       SPEC_SAFE_MATH|
+       SPEC_BIGINT_MATH|
+       SPEC_JAVA_MATH|
+       SPEC_PROTECTED|
+       SPEC_PUBLIC|
+       GHOST|
+       SPEC_NAME|  // ???
+       STATIC|
+       STRICTFP|
+       STRICTLY_PURE|
+       TWO_STATE|
+       UNINITIALIZED|
+       UNREACHABLE|
+       PURE|
+       RETURN_BEHAVIOR|
+       FINAL|
```

Comment:
I'd say: Java keyword and thus not allowed as identifier in JML
(same for assert, return, static, strictfp, const(?), native)